### PR TITLE
chore: omit VCS commands for changelog in favor of s3-only approach

### PR DESCRIPTION
**Pull request by codyshepherd**
Continuation of development on #19645 

Because `master` is a protected branch requiring at least 1 review of a PR for any push, we are instead electing to redirect those interested in the changelog to an s3 link instead of trying to update VCS with every changelog change.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The Changelog has moved!
+
+You can find it at the following URL:
+
+s3://dl.influxdata.com/platform/nightlies/master/CHANGELOG.md
```
Comment (lesam) [inline CHANGELOG.md:5]: Can we expose this over http (or something that you can type into the Chrome address bar at least?)

Comment (lesam) [reply CHANGELOG.md:5]: Also let's make it a markdown link please

Comment (codyshepherd) [reply CHANGELOG.md:5]: oh yes, we can, i'll fix

Comment (codyshepherd) [reply CHANGELOG.md]: Note that the link will be a 404 until the first changelog is uploaded successfully

**Review by lesam — APPROVED** [review]

**Pull request merged**